### ISSUE TITLE
Bias of Lya is too low

### ISSUE THREAD

**Issue by londumas** [igmhub/LyaCoLoRe#36]
The following plot gives the comparison between the bias of Lya in mocks (using the eBOSS option of quickquasar) and data. The fit to the data were done without metals, to go faster (I will update once I have them).

The parametric function is:
```
def alpha(z,a0,gamma,zeff=2.30):
    evol = a0*((1.+z)/(1.+zeff))**gamma
    return evol
```
we get:
```
Mocks:
a0 = -0.1544 +/- 0.0036
gamma = 3.10 +/- 0.37

Data:
a0 = -0.1819 +/- 0.0031
gamma = 2.44 +/- 0.26
```

We thus have a `~18%` difference between mocks and data.
This is a very big difference.

![bias_lya_mock_data](https://user-images.githubusercontent.com/3166436/48822615-eb744b80-ed1a-11e8-8595-642253b4f12f.png)

The following plot compares the two auto-correlation functions for one of the 4 wedges.

![auto_cf_data_mocks](https://user-images.githubusercontent.com/3166436/48802011-2353a280-eccb-11e8-9bb0-9c54e36fc5cb.png)

**Comment by londumas** [comment]
For what it is worth, here is the comparison for beta_Lya:
```
Mocks:
a0 = 1.367 +/- 0.063
gamma = -1.69 +/- 0.61

Data:
a0 = 1.12 +/- 0.033
gamma = -0.42 +/- 0.47
```
![beta_data_mocks](https://user-images.githubusercontent.com/3166436/48802186-ac6ad980-eccb-11e8-8e47-31684d14f50f.png)

**Comment by julienguy** [comment]
Helion, can you specify whether this is the true bias, or the picca "bias" parameter which is in fact bias*beta/f (and which should be renamed before causing more trouble) ? (I think it is the later from a comparison with DR12)

**Comment by londumas** [comment]
@julienguy, indeed it is the picca "bias" = bias*beta/f, that will have its name changed after all DR14 papers are submitted.
However it is the case for both data and mocks, so the comparison is ok.

**Comment by jfarr03** [comment]
@londumas how does this compare the the results at https://desi.lbl.gov/trac/wiki/LymanAlphaWG/LondonMocks? The picca "bias" should, I believe, correspond to "bias_eta(z)" in the first two plots on that page, at which point the bias values looks more reasonable. The plots there are without any noise from quickquasars, so it's not fair to compare the error bars, but the best fit values should still be a fair comparison? The values of beta are quite different though.

**Comment by londumas** [comment]
@jfarr03, could you send me the path to the `*.h5` of the correlations you used in https://desi.lbl.gov/trac/wiki/LymanAlphaWG/LondonMocks so I can add it on my plot?

**Comment by jfarr03** [comment]
Sure: they're at `/global/homes/j/jfarr/Programs/picca/picca_analysis_002/picca_?????/*.h5`. I've changed the permissions so hopefully you should be able to access them. The directories `picca_????` each refer to a run of picca in a specific redshift bin (see the `parameters.txt` file inside the directory for the list of parameters used). The .h5 filenames shows the minimum separation used and whether the alphas were left free or fixed to 1. E.g. rmin=20 and alphas free: `result_20r_afree.h5`. Let me know if that makes sense!

**Comment by jfarr03** [comment]
The bias/beta plots at the LondonMocks wiki page are made using rmin=40 and alphas free

**Comment by londumas** [comment]
@jfarr03, I can't read them. Could you run `fix_permissions.sh`?

**Comment by jfarr03** [comment]
@londumas It should work now. Failing that, how do I run fix_permissions?

**Comment by londumas** [comment]
@julienguy et al,
Plotting the real bias and the picca bias=bias_eta:

Real bias:

![bias_lya_mock_data](https://user-images.githubusercontent.com/3166436/48823296-8b32d900-ed1d-11e8-91bf-57ff1402dd5c.png)

bias picca = bias_eta

![bias_eta_lya_mocks](https://user-images.githubusercontent.com/3166436/48823297-8b32d900-ed1d-11e8-999f-c3c925eb7b9c.png)

**Comment by andreufont** [comment]
Hi @londumas - we should discuss this at the telecon today. A couple of comments:
 - In the first plots you talk about "gamma" as the redshift evolution of the "biases". However, once we allow "beta" to vary, we need a different "gamma" for "bias" and for "bias_eta", or a gamma for "beta". 
 - When comparing mocks vs data, we should include and HCD model in the data, otherwise we are measuring a combination of Lyman alpha bias and HCD absorption bias, it isn't really apples to apples. I think it is better to plot the values of bias/beta from your DR12 paper, i.e., from the right column of your Table 4, I read beta=1.65, b=-0.3544/(1+beta)=-0.134, b_eta= b beta /f = -0.227

**Comment by londumas** [comment]
Here is the status of this issue as of `london/v5.0.0/quick-0.0`:
- bias of lya is now the same in mocks and data
- beta of lya is quite different between mocks and data: data is around twice the mocks
- hence bias_eta is quite different between mocks and data: data is around twice the mocks

Thanks for the work on the bias of Lya @jfarr03, is there a way to work on the beta of Lya? 

![bias_eta_lya](https://user-images.githubusercontent.com/3166436/51492768-2a04a180-1d70-11e9-82a6-4c9244272c0c.png)

![beta_lya](https://user-images.githubusercontent.com/3166436/51492769-2a04a180-1d70-11e9-8c30-fef9fa1e731d.png)

![bias_lya](https://user-images.githubusercontent.com/3166436/51492767-2a04a180-1d70-11e9-987e-10bf082f1c6e.png)

**Comment by londumas** [comment]
Here is the status as of `v6.0/v6.0.<*>`.
The first two plots are for Lya absorption in Lya region, the second plot for Lya absorption in the Lyb region.

 ```
cf: <bias_eta_LYA> = -0.15894508333511256 +/- 0.0012263691674707807 (<0.0032192769916185235>, 0.0010180247712489323)
xcf: <bias_eta_LYA> = -0.15003992845706787 +/- 0.0017057947589636424 (<0.005711528011616922>, 0.0018061437436562116)
cfxcf: <bias_eta_LYA> = -0.1547722569156497 +/- 0.0008244683557990767 (<0.002701973829412652>, 0.0008544391479111237)

cf: <beta_LYA> = 1.2018049728610194 +/- 0.0157541871972593 (<0.040874734633205824>, 0.012925726019589752)
xcf: <beta_LYA> = 1.153946942947282 +/- 0.020393346170338156 (<0.06902682936536614>, 0.0218282000454352)
cfxcf: <beta_LYA> = 1.167835543631884 +/- 0.0109609361722942 (<0.033711922429322555>, 0.010660645917957602)

cf: <bias_eta_LYA> = -0.15572722071609243 +/- 0.003345130611980176 (<0.007386818468793355>, 0.002335917102358442)
xcf: <bias_eta_LYA> = -0.16557919460708742 +/- 0.0047303843193793275 (<0.016532095444068744>, 0.005227907609855003)
cfxcf: <bias_eta_LYA> = -0.1537895586919697 +/- 0.003100119166115423 (<0.00667899959452741>, 0.002112085121004769)

cf: <beta_LYA> = 1.2212425751039702 +/- 0.04135081132624239 (<0.1061094403188864>, 0.03355475126533844)
xcf: <beta_LYA> = 1.4232827004886694 +/- 0.07406440597097134 (<0.23927180832527917>, 0.07566438941751207)
cfxcf: <beta_LYA> = 1.2034199392843419 +/- 0.035848677295119216 (<0.09546753018456205>, 0.0301894837974091)
````

![bias_eta_lya_lyaforest](https://user-images.githubusercontent.com/3166436/55266101-e5e6bf00-5240-11e9-8ab7-429b7fdd55b0.png)
![beta_lya_lyaforest](https://user-images.githubusercontent.com/3166436/55266100-e5e6bf00-5240-11e9-9805-b9c74feb8cea.png)

![bias_eta_lya_lybforest](https://user-images.githubusercontent.com/3166436/55266103-e5e6bf00-5240-11e9-9a59-e0cad0876610.png)
![beta_lya_lybforest](https://user-images.githubusercontent.com/3166436/55266102-e5e6bf00-5240-11e9-9468-99ae765b2339.png)

**Comment by jfarr03** [comment]
@londumas thanks for all of these plots, they look great! 

I'm guessing that the horizontal lines correspond to averages of the different types of correlations? Would it be possible to match the colours of the lines to the points for the individual realisations? It's a minor thing but it'd make them easier to read I think! 

Could you also confirm what redshift these are measured at? And how many skewers are used to measure on? 

In terms of the actual values, it's clear that the value of b_eta (and subsequently beta) is still too low, as expected -  we haven't quite finalised how to fix this yet, but it's not too far off hopefully!

**Comment by londumas** [comment]
@jfarr03, if I understood correctly the last presentation, this is fixed. Could you add a plot and then we can close the ticket?

**Comment by jfarr03** [comment]
It's fixed to some extent, in that we know how to boost bias_eta. The results still don't match data exactly, but we haven't tried to match to data very precisely until we determine exactly what data we should match to (i.e. DR12 or 14)

**Comment by londumas** [comment]
Could you put some plots on the ticket? From what I have seen it is fixed, as far as a ticket goes.

**Comment by jfarr03** [comment]
@londumas sure good idea. Please find below:

Plots showing that the biases in v7.0.0 are close to BOSS DR12 values:
[biases_40r_afixed.pdf](https://github.com/igmhub/LyaCoLoRe/files/3193028/biases_40r_afixed.pdf)
[beta_biases_40r_afixed.pdf](https://github.com/igmhub/LyaCoLoRe/files/3193031/beta_biases_40r_afixed.pdf)

Evidently the match is not exact, but it can be improved upon with relative ease once the target values are chosen.

**Comment by londumas** [comment]
Good, do you have a plot of the 4 wedges of the correlation function: mock and data? We can end the ticket with that.

**Comment by jfarr03** [comment]
Indeed: 
![cf_40r_afixed_piccafit_v7 0 0](https://user-images.githubusercontent.com/15346565/57957200-68136b80-78f3-11e9-9755-1b5ad1e145db.png)

**Comment by londumas** [comment]
Fixed in v7.0

**Comment by londumas** [comment]
@jfarr03, thanks for the work.
